src/indexer/services/coinmarketcap/coinmarketcap.service.ts

```diff
@@ -35,12 +35,12 @@ class CoinMarketCapService {
   }
 
   private async getValueConvertion(amount: string, tokenSymbol: string): Promise<CoinMaketCapResponse["quote"]["USD"]["price"]> {
-    const tokenValue: string | undefined = await this.memoryCache.get(tokenSymbol)
+    const tokenValue: number | undefined = await this.memoryCache.get(tokenSymbol)
     if (tokenValue) {
-      return BigNumber(amount).times(BigNumber(tokenValue))
+      return Number(amount) * tokenValue
     }
 
-    const url = `${this.coinMarketCapUrl}/v2/tools/price-conversion?amount=${amount}&symbol=${tokenSymbol}&convert=USD`
+    const url = path.join(this.coinMarketCapUrl, `/v2/tools/price-conversion?amount=1&symbol=${tokenSymbol}&convert=USD`)
```

Comment:
What's the difference here :sweat_smile: ?

Reply:
amount=1 in the url. If we pass the actual value of the amount, the coinmarketcap automatically calculates the usdValue. But then we can't store it in memory to use it later to reduce the number of coinmarketcap API calls